tests/v2/actions/workloads/deployment/deployment.go

```diff
@@ -99,39 +100,35 @@ func UpdateDeployment(client *rancher.Client, clusterID, namespaceName string, d
 }
 
 // RolbackDeployment is a helper to rollback deployments
-func RollbackDeployment(client *rancher.Client, clusterID, namespaceName string, deployment *appv1.Deployment, revision int) (string, error) {
-	deploymentCmd := fmt.Sprintf("deployment.apps/%s", deployment.Name)
+func RollbackDeployment(client *rancher.Client, clusterID, namespaceName string, deploymentName string, revision int) (string, error) {
+	deploymentCmd := fmt.Sprintf("deployment.apps/%s", deploymentName)
 	revisionCmd := fmt.Sprintf("--to-revision=%s", strconv.Itoa(revision))
 	execCmd := []string{"kubectl", "rollout", "undo", "-n", namespaceName, deploymentCmd, revisionCmd}
 	logCmd, err := kubectl.Command(client, nil, clusterID, execCmd, "")
 	return logCmd, err
 }
 
-// GetRolloutHistoryDeployment is a helper to get rollout history deployment
-func GetRolloutHistoryDeployment(client *rancher.Client, clusterID, namespaceName string, deployment *appv1.Deployment) ([]string, error) {
-	deploymentCmd := fmt.Sprintf("deployment.apps/%s", deployment.Name)
-	execCmd := []string{"kubectl", "rollout", "history", "-n", namespaceName, deploymentCmd}
-	logCmd, err := kubectl.Command(client, nil, clusterID, execCmd, "")
-
+// CheckRolloutHistoryDeployment is a helper to get rollout history deployment
+func CheckRolloutHistoryDeployment(client *rancher.Client, clusterID, namespaceName string, deploymentName string, expectedRevision string) error {
```

Comment:
one last thing 😅 can you just name this to `ValidateRolloutHistoryDeployment`

Reply:
I can